modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/storage/constants/FieldConstants.java

```diff
@@ -106,9 +106,22 @@ public static Serializable getSerializable(
 			}
 
 			try {
-				Number number = numberFormat.parse(GetterUtil.getString(value));
+				String valueString = String.valueOf(
```

Comment:
Can we change the name of this variable? We already have a `value` that's of type `String`.

Reply:
Hey @rodrigopaulino, I agree with you but I can't figure out a better name 😅
Do you have any one in your mind?

Reply:
Maybe `formattedValue`. What do you think?

Reply:
I could replace simply replace `value` with ` String.valueOf(numberFormat.parse(GetterUtil.getString(value)))`. What do you think about it?